packages/cfd/src/features/Containers/trading-password-manager.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Text, Button, Icon, MultiStep, SendEmailTemplate } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { getCFDPlatformLabel } from '@deriv/shared';
+import { useVerifyEmail } from '@deriv/api';
+import ChangePasswordConfirmation from '../../Containers/cfd-change-password-confirmation';
+import { TChangePassword, TPasswordResetAndTradingPasswordManager } from '../../Containers/props.types';
+import { CATEGORY, CFD_PLATFORMS } from '../../Helpers/cfd-config';
+
+const ChangePassword = ({ platform, onConfirm }: TChangePassword) => (
+    <div className='cfd-change-password'>
+        <Icon
+            className='cfd-change-password__icon'
+            icon={platform === CFD_PLATFORMS.MT5 ? 'IcMt5OnePassword' : 'IcDxtradeOnePassword'}
+            width='122'
+            height='108'
+        />
+        <Text as='p' align='center' size='s' weight='bold'>
```

Comment:
no need to pass the size value here. i think its already s in the text component

Reply:
Good catch 👍